uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestTest.php

```diff
@@ -233,6 +235,47 @@ public function testDeleteSoftDeletesTheLeaveRequest() {
     $this->assertEquals(1, $leaveRequestRecord->is_deleted);
   }
 
+  public function testPublicHolidayLeaveRequestIsDeletedAndBalanceRecalculatedForOverlappingLeaveRequestDate() {
+    AbsencePeriodFabricator::fabricate([
+      'start_date' => CRM_Utils_Date::processDate('2016-01-01'),
```

Comment:
Do you think it might be a good idea to change these tests to use generated dates relative to the current date? It might bring up errors for edge cases when the tests are run on certain days.

Reply:
doesn't really matter, for this particular test, we are only concerned that a public holiday leave request is deleted for a particular date. 
For relative dates, we have tons of tests in the public holiday deletion service tests that already does justice to that.